xml/upnp_banners.xml

```diff
@@ -531,6 +545,38 @@
     <param pos="1" name="os.version"/>
     <param pos="0" name="os.cpe23" value="cpe:/o:apple:mac_os_x:{os.version}"/>
   </fingerprint>
+  <fingerprint pattern="^Loxone Miniserver (.*) UPnP/1.0$">
+    <description>Loxone Miniserver Smart Home</description>
+    <example host.name="some name">Loxone Miniserver some name UPnP/1.0</example>
+    <param pos="0" name="hw.vendor" value="Loxone"/>
+    <param pos="0" name="hw.product" value="Miniserver"/>
+    <param pos="0" name="hw.device" value="Management"/>
```

Comment:
`Management` seems vague. Perhaps `Smart Home Management`, `Home Automation`, etc?

Reply:
Yes, I had the same hesitation.  I think "Building Automation" or similar might work better given other work

Reply:
Building Automation sounds like  a good compromise.